--- src/orm/schema/diff_schema.rs	2025-08-11 01:20:06
+++ src/orm/schema/orm_schema.rs	2025-08-11 01:48:14
@@ -1,12 +1,10 @@
 // @generated automatically by Diesel CLI.
 
 pub mod sql_types {
-    #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
+    pub use postgis_diesel::sql_types::Geometry;
 
     #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "object_direction_definition"))]
     pub struct ObjectDirectionDefinition;
 
     #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
